lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -219,7 +219,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_indonesia
 
-      outcome :outcome_os_laos
+      outcome :outcome_opposite_sex_marriage_laos_with_lao_national
```

Comment:
I've rewritten the branch and followed your suggestion in 55616f117b1bb7744d77d27136db207824d314a6.